.drone.yml

```diff
@@ -1846,10 +1846,9 @@ steps:
     done
   environment:
     RELEASES_CERT:
-      from_secret: RELEASES_CERT_STAGING
+      from_secret: RELEASES_CERT
     RELEASES_KEY:
-      from_secret: RELEASES_KEY_STAGING
-  failure: ignore
```

Comment:
If we stop ignoring failures, doesn't this mean that every subsequent rerun of the tag build will fail? Right now "publish to release service" step fails every time we re-run tag build (e.g. due to a failure).

Reply:
Yes. The solution for now is to manually destroy the partially-created release, and then retry the build. We have [created a runbook for this](https://github.com/gravitational/cloud/blob/master/docs/runbooks/releases.md#destroy-a-release) and noted this in the [release checklist](https://gravitational.slab.com/posts/teleport-release-checklist-cf0s6vmt#h9b88-release-process) (Ctrl-F "If the tag build failed").

Reply:
What is the plan to address this limitation and what is the ETA?

I publish 99% of Teleport releases and it's already a pretty involved process so I would really love if there was no yet another thing that I have to do manually.

Reply:
One straightforward solution comes to mind - perhaps Drone could execute this "destroy" workflow before reretrying the tag build. I'll try to prototype this ASAP, and if it works, perhaps we can slip in that change before this PR.

Reply:
@r0mant Please see https://github.com/gravitational/teleport/pull/16669 . If the solution is sufficient, I will try to rebase / remake this PR on top of it.